src/controller/pagination_controller.rs

```diff
@@ -16,8 +16,7 @@ pub async fn pagination_display(
     let posts_per_page = total_posts_length / 3.0;
     let posts_per_page = posts_per_page.round();
     let posts_per_page = posts_per_page as usize;
-    let pages_count :Vec<_>= (1..=posts_per_page).into_iter().collect();
-
+    let pages_count: Vec<_> = (1..=posts_per_page).collect();
```

Comment:
We don't need this `pages_count` and simply assigning `1..=posts_per_page` in `json` will do
```suggestion
    let htmls = handlebars.render("admin_page", &json!({..."pages_count":1..=posts_per_page, ...}))
        .map_err(...)?;
```